[2022-12-30 12:47:52,038] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: insert_new_rates.dag2_extract_rates manual__2022-12-30T12:47:50.502629+00:00 [queued]>
[2022-12-30 12:47:52,043] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: insert_new_rates.dag2_extract_rates manual__2022-12-30T12:47:50.502629+00:00 [queued]>
[2022-12-30 12:47:52,043] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-30 12:47:52,043] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-30 12:47:52,043] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-30 12:47:52,054] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): dag2_extract_rates> on 2022-12-30 12:47:50.502629+00:00
[2022-12-30 12:47:52,059] {standard_task_runner.py:52} INFO - Started process 3803 to run task
[2022-12-30 12:47:52,061] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'insert_new_rates', 'dag2_extract_rates', 'manual__2022-12-30T12:47:50.502629+00:00', '--job-id', '71', '--raw', '--subdir', 'DAGS_FOLDER/insert_new_rates.py', '--cfg-path', '/tmp/tmpqtgi_btd', '--error-file', '/tmp/tmpb2e2br5f']
[2022-12-30 12:47:52,062] {standard_task_runner.py:80} INFO - Job 71: Subtask dag2_extract_rates
[2022-12-30 12:47:52,097] {task_command.py:369} INFO - Running <TaskInstance: insert_new_rates.dag2_extract_rates manual__2022-12-30T12:47:50.502629+00:00 [running]> on host 99c1092660ad
[2022-12-30 12:47:52,153] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=insert_new_rates
AIRFLOW_CTX_TASK_ID=dag2_extract_rates
AIRFLOW_CTX_EXECUTION_DATE=2022-12-30T12:47:50.502629+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-30T12:47:50.502629+00:00
[2022-12-30 12:47:52,744] {python.py:173} INFO - Done. Returned value was: {
    "success": false,
    "error": {
        "code": 502,
        "type": "invalid_start_date",
        "info": "You have specified an invalid start date. Please try again or refer to the API documentation available at https://serpstack.com/documentation."
    }
}

[2022-12-30 12:47:52,762] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=insert_new_rates, task_id=dag2_extract_rates, execution_date=20221230T124750, start_date=20221230T124752, end_date=20221230T124752
[2022-12-30 12:47:52,797] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-30 12:47:52,823] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
